packages/protocol/contracts-0.8/common/GasPriceMinimum.sol

```diff
@@ -53,7 +56,7 @@ contract GasPriceMinimum is
    * @return Patch version of the contract.
    */
   function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
-    return (1, 2, 0, 0);
+    return (1, 2, 0, 1);
```

Comment:
The tooling may complain because it won't look up contracts from the previous release that are in the contracts-8 folder.

See https://github.com/celo-org/celo-monorepo/issues/10567 and https://github.com/celo-org/celo-monorepo/issues/10628